src/components/Tabs/index.tsx

```diff
@@ -0,0 +1,27 @@
+import styles from '@/components/Tabs/Tabs.module.scss';
+import { TABS } from '../tasks/constants';
+
+type TabsProps = {
+  tabs: typeof TABS;
+  onSelect: (tab: string) => void;
+  activeTab: string
+};
+
+export default function Tabs({ tabs, onSelect, activeTab }: TabsProps) {
+
+  const tabButtons = tabs.map((tab: string, index: number) => (
```

Comment:
`[FOR NEXT TIME] Suggestion:` You can write this logic inside the return function as well. 
Use this style of code for early or conditional returned JSX.